ocean_lib/ocean/mint_fake_ocean.py

```diff
@@ -14,10 +14,11 @@
 from ocean_lib.web3_internal.utils import from_wei, get_ether_balance
 from ocean_lib.web3_internal.wallet import Wallet
 from ocean_lib.web3_internal.web3_provider import Web3Provider
-from tests.resources.helper_functions import (
-    get_ganache_wallet,
-    get_publisher_ocean_instance,
-)
+
```

Comment:
this is part of another PR. Please keep things separate for the purpose of simple code review.